packages/catalog-server/src/lib/firestore/custom-element-converter.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  DocumentData,
+  FirestoreDataConverter,
+  QueryDocumentSnapshot,
+} from '@google-cloud/firestore';
+
+import {CustomElement} from '@webcomponents/catalog-api/lib/schema.js';
+
+export const customElementConverter: FirestoreDataConverter<CustomElement> = {
+  fromFirestore(snapshot: QueryDocumentSnapshot<DocumentData>): CustomElement {
+    return {
+      package: snapshot.get('package'),
+      version: snapshot.get('version'),
+      // TODO: convert from Map to list
+      distTags: snapshot.get('distTags'),
+      author: snapshot.get('author'),
+      tagName: snapshot.get('tagName'),
+      className: snapshot.get('className'),
+      customElementExport: snapshot.get('customElementExport'),
+      declaration: snapshot.get('declaration'),
+    };
+  },
+  toFirestore(_packageInfo: CustomElement) {
+    throw new Error('not implemented');
```

Comment:
Why doesn't this need to be implemented?

Reply:
I'm in the middle of more consistently using converters for both read and write operations. I currently don't use them for writing custom elements, but wanted to throw here in case it was used that way so we wouldn't just drop all the data.